openhtf/io/user_input.py

```diff
@@ -24,14 +24,20 @@
 
 import collections
 import functools
+import logging
+import platform
 import select
 import sys
-import termios
 import threading
 import uuid
 
 import gflags
 
+if platform.system() != 'Windows':
+  import termios
```

Comment:
mind if we make this a separate feature request? it's a nontrivial amount of work and i'd like to get this in.